apps/explorer/src/components/transactions/ProgTxnBlockCard.tsx

```diff
@@ -12,20 +12,26 @@ interface ProgrammableTxnBlockCardProps {
 	itemsLabel: string;
 	defaultItemsToShow: number;
 	noExpandableList?: boolean;
+	itemsLabelPrefix?: string;
```

Comment:
The labels are used in the `expand/collapse` section also. If I change it to `2 items`, when there's a collapsible section it will render as `show 2 2 items`

<img width="879" alt="Screenshot 2023-08-02 at 11 20 18 AM" src="https://github.com/MystenLabs/sui/assets/127577476/a246add4-7fdf-463e-9987-3ce8a29af537">

Reply:
Ah I see. I'd just make this like `count` then to make it more obvious

Reply:
updated